main.go

```diff
@@ -66,5 +67,19 @@ func main() {
 		}
 	}
 
-	os.Exit(0)
+	if *context.CliArguments.Mode == cliparser.ConfigureMode {
+		if !configurevalidator.GetExist() {
+			configurevalidator.ShowRegions()
+			configurevalidator.MakeMapRegion()
+			configurevalidator.MakeYaml()
+		}
+			valid := offlinevalidator.Validate(&context)
```

Comment:
We do not know if user wants to validate offline or online in this point. Just return success here and do not perform validation.